extension/resources/web-components-payment-type.json

```diff
@@ -27,6 +27,28 @@
       "inputHint": "SingleLine",
       "required": false
     },
+    {
+      "name": "getPaymentMethodsRequest",
+      "label": {
+        "en": "getPaymentMethodsRequest"
+      },
+      "type": {
+        "name": "String"
+      },
+      "inputHint": "MultiLine",
+      "required": false
+    },
+    {
+      "name": "getPaymentMethodsResponse",
+      "label": {
+        "en": "getPaymentMethodsResponse"
+      },
+      "type": {
+        "name": "String"
+      },
+      "inputHint": "MultiLine",
+      "required": false
+    },
     {
       "name": "makePaymentRequest",
```

Comment:
Do we still need this?

Reply:
Yes. Otherwise Adyen API request and response cannot be put into payment custom fields.

By the way, I removed obsoleted fields from this JSON
https://github.com/commercetools/commercetools-adyen-integration/pull/1075/commits/84dbf1e1bdb153ed349ce6cc2864b777dd223e74